docs/mpe.md

```diff
@@ -3,47 +3,51 @@
 
-* Distances: The landmarks and agents typically start out uniformly randomly placed from -1. to 1. on the map. This means they are typically around 1-2 units distance apart, so keep that in mind when reasoning about the scale of the rewards, which often depend on distance, and the observation space, which contain relative and absolute positions.
+* **Distances**: The landmarks and agents typically start out uniformly randomly placed from -1 to 1 on the map. This means they are typically around 1-2 units apart. This is important to keep in mind when reasoning about the scale of the rewards (which often depend on distance) and the observation space, which contains relative and absolute positions.
 
 ### Termination
 
-The game terminates after a number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original source code, the default value was 25, not 100.
+The game terminates after the number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original OpenAI source code, the default value was 25, not 100.
 
 ### Observation Space
 
-The observation space of an agent is a vector generally composed of the agent's position and velocity, other agent's relative position and velocity, the landmarks relative positions, the landmark's and agent's types, and communications it received from other agents. The exact form of this is detailed in the environments.
+The observation space of an agent is a vector generally composed of the agent's position and velocity, the other agents' relative positions and velocities, the landmarks' relative positions, the landmarks' and agents' types, and communications received from other agents. The exact form of this is detailed in the environments' documentation.
 
-If an agent_1 cannot see or observe the communication of agent_2, then agent_2 is not included in agent_1's observation space, resulting in varying observation space sizes in certain environments.
+If an agent cannot see or observe the communication of a second agent, then the second agent is not included in the first's observation space, resulting in varying observation space sizes in certain environments.
 
 ### Action Space
 
-The action space is a discrete action space representing the combinations of the movements and communications an agent can perform. Agents that can move can choose between the 4 cardinal directions and do nothing. Agents that can communicate choose between 2 and 10 environment dependent options, and the message is broadcast all agents which can hear it.
+The action space is a discrete action space representing the combinations of movements and communications an agent can perform. Agents that can move can choose between the 4 cardinal directions or do nothing. Agents that can communicate choose between 2 and 10 environment dependent broadcasting options.
```

Comment:
Revert the change at the end of the last sentence.

Reply:
Ok this one is still confusing to me. Agents who can move have 4 options. Can agents who communicate move? Are those movement options included in the 2-10 environment dependent options? Or are they incapable of moving, with 2-10 communication options? The last part is confusing because "the message" hasn't been referenced anywhere yet.

Reply:
My complaint is that not all agents can hear other agents.

Can agents who communicate move?  Depends on the agent type and game

Are those movement options included in the 2-10 environment dependent options? what?

Or are they incapable of moving, with 2-10 communication options? Again, what?

The last part is confusing because "the message" hasn't been referenced anywhere yet. Maybe go with the old one with a better word than message?

Reply:
"Agents that can communicate choose between 2 and 10 environment-dependent communication options, which broadcast a message to all agents that can hear it."?

Reply:
sounds good

Reply:
2 -10 is the number of communication options offered by the environment. 

To make this more clear, perhaps: Agents that can communicate choose a discrete value to broadcast to all agents that can hear it. The number of values they can choose from varies between 2 to 10, dependent on the environment.